# This contains a list of Odometry type messages
nav_msgs/Odometry[] array